pitch.md

```diff
@@ -23,38 +79,101 @@ should be more specific than "The user describes properties of unicorns.", e.g.,
 
 ## Related work
 _Are there any other DSLs in this domain? If not, describe how you know there
 aren't and conjecture why not. If so, describe them and provide links. How well 
 do they address the need? Are there any particularly admirable qualities of the
 language? Are there parts of the language you think could be improved?_
 
+The only information I could find is that to control these lights you must use 
+Arduino or Raspberry Pi code to control the wire outputs for the LEDs. I would 
+not consider thisa  DSL in my domain but rather a DSL in the domain of all 
+electrical engineering projects. I think the reason why there hasnt been a DSL 
+is either because those interested in the domain are okay using the regular 
+Arduino code or because it is quite a specific domain so it doesnt seem likely 
+to have languages created for it. 
+
+Here is a link to the arduino library for addressable LEDs 
+https://learn.adafruit.com/adafruit-neopixel-uberguide/arduino-library 
+which really shows the complexity of the language. 
+
 
 ## Suitability
 _What percentage of your time do you think will be spent directly engaging in
 the **language** aspects of this project (e.g., making language design
 decisions), as opposed to "systems" aspects of the project (e.g., implementing a
 complicated semantics that doesn't require a lot of language design)?_
 
+I think that a lot (most maybe) of this project will be working on the **
+language** aspects. Once the back end methods of creating a fade function/color 
+change function/etc.. are completed, the rest will be trying to design the 
+language in a way that promotes chaining commands and easily integrates aspects 
+of looping and time to allow for complex light effects. I have already come up 
+with a few options but picking the right one is difficult because there are 
+lots of trade-offs to think of. 
+
 
 ## Scope
 _How big an idea is this? How ambitious is this project?_
 
+This project could be quite ambitious. There are lots of features that could be 
+made for the user and it is easy to get lost in "feature creating land" instead 
+of language design land. However, I also think the scope is not too large since 
+it focuses on a relatively specific domain. 
 
 ## Challenges and opportunities
 _Why is this a good project? What are some challenges you expect to face? How
 might you overcome them? What are some warning signs that the project has gotten
 off track, and how will you get the project back on track if needed?_
+
+This could be a great idea for a project since it is very fun, allows for a lot 
+of language design choices, and is mainly designed around allowing the user to 
+create complex creations out of simple code. This might not be a good idea 
+because it technically requires working with Arduino/Raspberry pi (even though 
+this part could realistically be ignored) and because it might be too specific 
+of a domain that it really is just useful for me and not a whole lot of others. 
+
```

Comment:
I think your concerns are valid, but I also think there is a lot of potential to this DSL. A DSL is supposed to be a language tailored to a specific group of people and make life easier. Overall, I really like this idea!